test/config/config-tracing.yaml

```diff
@@ -16,9 +16,9 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: config-tracing
-  namespace: knative-eventing
+  namespace: ${SYSTEM_NAMESPACE}
```

Comment:
with the `sed` that's a good one :smile: 

I will do the same for the consolidated channel too

Reply:
I originally used "envsubst" which worked perfectly locally but is apparently not available on the build image.